TCL/polarDensity_for_DTA.tcl

```diff
@@ -612,7 +632,7 @@ proc polarDensityBin { config_file_script } {
         set upp_f [open "${outfile}.upp.dat" w]
         set low_f_avg [open "${outfile}.low.avg.dat" w]
         set upp_f_avg [open "${outfile}.upp.avg.dat" w]
-        set totals [frame_leaflet_assignment "resname $species" $headname $tailname $lipidbeads_selstr $params(start_frame) $params(start_frame)]
```

Comment:
This switches the expected density measurement to use the end_frame instead of the start_frame